### PR TITLE
API Remove deprecated code

### PR DESCRIPTION
**Pull request by sabina-talipova**
### Description
- Legacy GraphQL classes were removed;
- Deprecated code was removed;
- Test cases / classes updated;

### Related issue
- https://github.com/silverstripe/silverstripe-framework/issues/10501

### CODE REVIEW

File: tests/GraphQL/FakeResolveInfo.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace DNADesign\Elemental\Tests\GraphQL;
 
+use ArrayObject;
+use GraphQL\Language\AST\OperationDefinitionNode;
```
Comment (sabina-talipova) [inline tests/GraphQL/FakeResolveInfo.php]:
```
use ArrayObject;
use GraphQL\Language\AST\OperationDefinitionNode;
```
Required types of parameters in ResolveInfo constructor : https://github.com/webonyx/graphql-php/blob/master/src/Type/Definition/ResolveInfo.php#L123-L132

**Comment by sabina-talipova** [comment]
Broken Behat tests are related to the issue https://github.com/silverstripeltd/product-issues/issues/655

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Merge conflict on FakeResolveInfo

Please rebase on 5 and use the version in this commit - https://github.com/silverstripe/silverstripe-elemental/commit/43dde3acd856557ef4e24b17c91f48eeae132db7#diff-445b2b99e2d2c31b3d1a34241221643396fd9134c296395559728b3f67ade3a0

**Comment by sabina-talipova** [comment]
DONE

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**